### DISCUSSION TITLE
CSS not delivered during first render after Next upgrade to 9.4.4

### DISCUSSION THREAD

**Discussion by Ziker22** [vercel/next.js#14932] [Help]
Hi all,
I have upgraded my project dependencies
One commit that updated react, react-dom and next introduced this bug when on first render SSR delivers content without CSS
Im using styled-jsx which remained untouched at version 3.3.0
Next version went from 9.2.0 --> 9.4.4
react/dom went from 16.12.0 -->16.13.1

I wonder if this is not known bug. But found nothing among github issues

Any help or ideas will be appreciated

EDIT : By error and trial it looks like this happens between 9.2.2 -->9.3.0

**Comment by jamesmosier** [comment]
This could potentially be related: https://github.com/vercel/next.js/issues/13058#issuecomment-647546667

Try upgrading to the latest canary (`9.4.5-canary.29`) to see if that helps

**Reply by Ziker22** [reply]
Thx for suggestion 
After naive upgrade im getting
```
"Cannot read property 'concat' of undefined","name":"TypeError","stack":"TypeError: Cannot read property 'concat' of undefined\n    at Object.webpack (C:\\xxx\\node_modules\\next-transpile-modules\\src\\next-transpile-modules.js:102:73)\n    at Object.webpack (C:\\xxx\\node_modules\\@next\\bundle-analyzer\\index.js:17:27)\n    at getBaseWebpackConfig (C:\\xxx\\node_modules\\next\\build\\webpack-config.ts:1014:28)\n    at async Promise.all (index 0)\n    at HotReloader.start (C:\\xxx\\node_modules\\next\\server\\hot-reloader.ts:312:21)\n    at DevServer.prepare (C:\xxx\\node_modules\\next\\server\\next-dev-server.ts:274:5)"},"msg":"Unhandled Promise Rejection","time":"2020-07-07T19:01:20.560Z","v":0}
```

This is problematic line in n-t-m module
` if (nextCssLoader) {
            nextCssLoader.issuer.include = nextCssLoader.issuer.include.concat(includes);`

Upgrading `next-transpile-modules` didnt help. Ill have to investgate this tommorow.

**Reply by jamesmosier** [reply]
Looks like a known issue (well, future issue since this is a canary Next build) https://github.com/martpie/next-transpile-modules/issues/88

Looks like a PR is out there for it: https://github.com/martpie/next-transpile-modules/pull/89

**Comment by Ziker22** [comment]
Ah yeah will have to wait for this to resolve https://github.com/martpie/next-transpile-modules/issues/88
